tigervnc/xfce4/xfconf/xfce-perchannel-xml/displays.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<channel name="displays" version="1.0">
+  <property name="ActiveProfile" type="string" value="Default"/>
+  <property name="Default" type="empty">
+    <property name="VNC-0" type="string" value="VNC-0">
+      <property name="Active" type="bool" value="true"/>
+      <property name="EDID" type="string" value=""/>
+      <property name="Resolution" type="string" value="3840x2160"/>
```

Comment:
So I'm on a 1x monitor, and when I first open it looks like this:
![Screen Shot 2020-10-20 at 10 47 50 AM](https://user-images.githubusercontent.com/649992/96611228-10d0b200-12c2-11eb-8022-90fba61a77a1.png)

When I enable remote resizing, it still looks like it's in 2x dpi mode:
![Screen Shot 2020-10-20 at 10 48 44 AM](https://user-images.githubusercontent.com/649992/96611285-1b8b4700-12c2-11eb-8218-9ea35a5c66c6.png)

Is there anything we can do to have it be more intelligent about the sizing?

Reply:
I've updated the URL parameters to use [scaling by default](https://github.com/cdr/c/pull/5798/files#diff-8674dd40465bbea9cd88ce8f97580cfe942dfab2119753f1d4c161d3f6a46999R45-R46), though I think we'll need to follow up with a resolution/DPI picker in the client to really make this comfortable.